internals/webpack/webpack.base.js

```diff
@@ -70,20 +70,24 @@ module.exports = (options) => ({
         NODE_ENV: JSON.stringify(process.env.NODE_ENV),
       },
     }),
-  ]),
-  postcss: () => [
-    postcssFocus(), // Add a :focus to every :hover
-    cssnext({ // Allow future CSS features to be used, also auto-prefixes the CSS...
-      browsers: ['last 2 versions', 'IE > 10'], // ...based on this browser list
-    }),
-    postcssReporter({ // Posts messages from plugins to the terminal
-      clearMessages: true,
+    new webpack.LoaderOptionsPlugin({
+      options: {
+        context: __dirname,
```

Comment:
Add comment above saying this is needed for `css-loader` and linking to the relevant issue in the webpack repo.